src/main/java/net/atos/zac/authentication/LoggedInUser.java

```diff
@@ -15,6 +15,8 @@ public class LoggedInUser extends User {
 
     private final Set<String> groupIds;
 
+    public static final String ROLE_FUNCTIONELE_GEBRUIKER = "functionele_gebruiker";
```

Comment:
It's also already defined (like all roles) in `rollen.rego` of course but I guess we cannot easily reuse that in our Java/Kotlin code? If not, good to add a little doc here that explains that this is a redefinition of this role name defined in `rollen.rego`?

Reply:
Probably good to add some doc to this particular role anyway as it is very specific. It is not one of our 4 standard roles but a specific 'system role' or whatever we want to call it.

Reply:
Added a small comment hinting this is `system role` used in async operations that have no access to the logged-in user